ROSCO_toolbox/utilities.py

```diff
@@ -153,17 +156,13 @@ def write_DISCON(turbine, controller, param_file='DISCON.IN', txt_filename='Cp_C
     file.write('{}              ! WE_FOPoles        - First order system poles [1/s]\n'.format(''.join('{:<10.8f} '.format(rosco_vt['WE_FOPoles'][i]) for i in range(len(rosco_vt['WE_FOPoles'])))))
     file.write('\n')
     file.write('!------- YAW CONTROL ------------------------------------------------------\n')
-    file.write('{:<13.5f}       ! Y_ErrThresh		- Yaw error threshold. Turbine begins to yaw when it passes this. [rad^2 s]\n'.format(rosco_vt['Y_ErrThresh']))
+    file.write('{:<13.5f}       ! Y_uSwitch		- Integrator saturation (maximum signal amplitude contribution to pitch from yaw-by-IPC), [rad]\n'.format(rosco_vt['Y_uSwitch']))
+    file.write('{}! Y_ErrThresh    - Yaw error threshold. Turbine begins to yaw when it passes this. If Y_uSwitch is zero, only the first value is used. [deg].\n'.format(''.join('{:<4.6f}  '.format(rosco_vt['Y_ErrThresh'][i]) for i in range(len(rosco_vt['F_FlCornerFreq'])))))
```

Comment:
Are there multiple values here? If so, what is the second value?

Reply:
There can be two values for `Y_uSwitch`. I fixed the `Y_ErrThresh` description to describe what happens. 

If `Y_uSwitch` > 0, the first value of `Y_ErrThresh` will be used for u < `Y_uSwitch`, and the second value will be used for u >= `Y_uSwitch`.